site/client/index.js

```diff
@@ -28,6 +29,10 @@ export function makeApp({ element, state }) {
     });
   }
 
+  // init Google Analytics tracker and publish a page view at '/'
+  ReactGA.initialize('UA-16654919-1');
```

Comment:
I do prefer this pattern, but it means we'll be initialising this script quite a lot later than we were before. This is probably ok but should keep an eye on it.